ui/component/commentCreate/view.jsx

```diff
@@ -11,11 +11,9 @@ import * as ICONS from 'constants/icons';
 import * as KEYCODES from 'constants/keycodes';
```

Comment:
After typing some comments in livestream, the HOME and END key does not bring the cursor to the start and end of the line.  Reverted the PR, then it'll work.